GVRf/Sample/gvr-video/src/org/gearvrf/video/VideoScript.java

```diff
@@ -632,6 +632,38 @@ public void run() {
             mainScene.getMainCameraRig().getOwnerObject()
                     .addChildObject(mPassThroughObject);
 
+            /*
+             * Selective objects to keep
```

Comment:
I also thought about this, it would be code purpose versus code neatness. This object ref count mechanism even can be changed in the future. 
For now, neatness seems more valued, and let me change like that as Jon’s suggestion.